package.json

```diff
@@ -31,6 +31,7 @@
     "glob": "^7.1.3",
     "jest": "^24.6.0",
     "lerna": "^3.13.1",
+    "metro-memory-fs": "^0.53.1",
```

Comment:
Please move it to `devDependencies`, it's only used for testing, no need to feed it to the users